src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -263,6 +265,16 @@ public async override System.Threading.Tasks.Task RunTaskAsync ()
 				}
 			}
 
+			if (AdditionalAndroidAssetPaths != null) {
+				for (int i = AdditionalAndroidAssetPaths.Length - 1; i >= 0; i--) {
```

Comment:
Why go through `@(LibraryAssetDirectories)` *backwards*? Do later `-A PATH` options "take precedence" over earlier ones?

Reply:
We go through backwards like we do for other resources. I assume its like the `-R` option where ` The last conflicting resource given takes precedence.`. I will have to test it.

Reply:
First Item takes precedence it seems. So I switches the loop order and also made sure we add the main app assets first.